stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4623 @@
+from _typeshed import Incomplete
+from collections.abc import Sequence
+from typing import Any, ClassVar, TypeVar, Union, overload
+from typing_extensions import TypeAlias
+
+from cv2.gapi.streaming import queue_capacity
+
+# #5768
+# import numpy
+_Mat: TypeAlias = Incomplete  # numpy.ndarray[int, np.dtype[np.generic]]
+_MatF: TypeAlias = Incomplete  # numpy.ndarray[float, np.dtype[np.generic]]
+
+# Function argument types
+_NumericScalar: TypeAlias = float | bool | None
+_Scalar: TypeAlias = _Mat | _NumericScalar | Sequence[_NumericScalar]
+_Point: TypeAlias = Union[tuple[int, int], Sequence[int]]  # noqa: Y047
+_Size: TypeAlias = Union[tuple[int, int], Sequence[int]]  # noqa: Y047
+_Range: TypeAlias = Union[tuple[int, int], Sequence[int]]  # noqa: Y047
+_PointFloat: TypeAlias = Union[tuple[float, float], Sequence[float]]  # noqa: Y047
+_SizeFloat: TypeAlias = Union[tuple[float, float], Sequence[float]]  # noqa: Y047
+_Rect: TypeAlias = Union[tuple[int, int, int, int], Sequence[int]]  # noqa: Y047
+_Boolean: TypeAlias = bool | int | None  # noqa: Y047
+# _UMat also covers InputArray and InputOutputArray
+_UMat: TypeAlias = UMat | _Mat | _NumericScalar
+_UMatF: TypeAlias = UMat | _MatF | _NumericScalar
+
+_TUMat = TypeVar("_TUMat", bound=_UMat)  # noqa: Y018
```

Comment:
Remind me which one Y018 is (😅) and why we're `noqa`ing it?

Reply:
Y047 flags the unused TypeAlias, and Y018 the unused TypeVar. In both cases they're reused elsewhere in the stub. And they will certainly be used in this file once enough types are filled in.

Why here instead of `cv2/__init__.pyi`? Because `cv2` is just a re-export of `cv2.cv2` for convenience.